aten/src/ATen/native/cpu/ReduceOpsKernel.cpp

```diff
@@ -174,61 +174,72 @@ static void norm_kernel_tensor_iterator_impl(
   if (p.isIntegral(false)) {
     val = p.to<int64_t>();
   } else if (p.isFloatingPoint()) {
-    val = p.to<float>();
+    val = p.to<double>();
```

Comment:
Why the change from float to double here?

Reply:
To me, it seems better to cast to double to allow more precision for `p`.